src/field/field_factory.cxx

```diff
@@ -97,7 +97,13 @@ FieldFactory::~FieldFactory() {
 }
 
 const Field2D FieldFactory::create2D(const string &value, Options *opt, Mesh *m, CELL_LOC loc, BoutReal t) {
```

Comment:
While you're fixing this, can we also change `m` to something like `localmesh`?

Ideally, we'd rename the global `mesh` to `global_mesh`, and then we could just use plain `mesh` everywhere